sympy/physics/mechanics/kane.py

```diff
@@ -23,26 +25,34 @@ class KanesMethod(object):
     equations of motion in the way Kane presents in:
     Kane, T., Levinson, D. Dynamics Theory and Applications. 1985 McGraw-Hill
 
-    The attributes are for equations in the form [M] udot = forcing.
+    The dynamical equations of motion in the form::
+
+        mass_matrix * speeds.diff() = forcing
+
+    where the mass matrix, M, and the right hand side vector, f, are available
+    as class attributes after Kane's equations are formed.
 
     Attributes
     ==========
 
-    q, u : Matrix
-        Matrices of the generalized coordinates and speeds
-    bodylist : iterable
-        Iterable of Point and RigidBody objects in the system.
-    forcelist : iterable
+    q, coordinates : Matrix, shape(n, 1)
+        Matrices of the generalized coordinates.
+    u, speeds : Matrix, shape(n, 1)
+        Matrices of the generalized speeds. For a non-holonomic system these
+        are ordered as [u_s, u_r].
```

Comment:
s: independent coordinates/speeds
r: dependent coordinates/speeds

This is the notation from Kane's book. I was think about using it in the math and internally to the class.